web/src/main/webapp/table.ts

```diff
@@ -497,6 +498,7 @@ export class TableView extends RemoteTableObjectView
                 this.contextMenu.addItem({text: "PCA...", action: () => this.pca() });
                 this.contextMenu.addItem({text: "Filter...", action: () => this.equalityFilter(cd.name)});
                 this.contextMenu.addItem({text: "Convert...", action: () => ColumnConverter.dialog(cd.name, TableView.allColumnNames(this.schema), this)});
+		this.contextMenu.addItem({text: "Count Distinct Elements", action: () => this.hLogLog(cd.name)});
```

Comment:
"Estimate", not "count".
Also, please keep the menu items without dots above the ones with dots.